filesystem.go

```diff
@@ -63,7 +69,10 @@ func CopyTree(sourcetree, desttree string) error {
 		target := path.Join(desttree, suffix)
 		switch info.Mode() & os.ModeType {
 		case 0:
-			CopyFile(p, target, info.Mode())
+			err := CopyFile(p, target, info.Mode())
+			if err != nil {
+				log.Panicf("Failed to copy file %s: %v", p, err)
```

Comment:
This should return err instead of panicking; potentially extending the error message to make it more clear what when wrong